src/java.base/share/classes/java/lang/classfile/CodeTransform.java

```diff
@@ -81,11 +85,22 @@ public void atEnd(CodeBuilder builder) {
         };
     }
 
+    /**
+     * @implSpec
+     * The default implementation returns this code transform chained with another
+     * code transform from the argument. Chaining of two transforms requires to
+     * involve a chained builder serving as a target builder for this transform
+     * and also as a source of elements for the downstream transform.
+     */
     @Override
     default CodeTransform andThen(CodeTransform t) {
         return new TransformImpl.ChainedCodeTransform(this, t);
     }
 
+    /**
+     * @implSpec The default implementation returns a resolved transform;
```

Comment:
`... returns a resolved transform that is bound to the given code builder`
or
`... returns a resolved transform bound to the given code builder`

Reply:
Fixed, thanks.